tasks/prettier_conformance/snapshots/prettier.ts.snap.md

```diff
@@ -61,8 +58,6 @@ ts compatibility: 532/598 (88.96%)
 | typescript/prettier-ignore/mapped-types.ts | 💥 | 96.61% |
 | typescript/property-signature/consistent-with-flow/comments.ts | 💥 | 80.00% |
 | typescript/property-signature/consistent-with-flow/union.ts | 💥 | 85.71% |
-| typescript/type-arguments-bit-shift-left-like/3.ts | 💥 | 0.00% |
```

Comment:
```ts
(class extends f<<T>(v: T) => void> {});
```

TSC also reports error for this.

https://www.typescriptlang.org/dev/bug-workbench/?ignoreDeprecations=6.0#code/PTAEAEEsHMDsHsBOBTAIsgDigxgQwC6TywDOAXKAGwB0ADAFD0gQBmkANsrLgLbIW5q+EvQAU2drhIlQyAB74uAExksAPGoAqAPlEA3CpoCUoALzbQe+JCUWA3gF8jAbnpA